nightmare_firmware/servos.h

```diff
@@ -5,21 +5,57 @@
 #include "Arduino.h"
 #include "definitions.h"
 
+/**
+ * @brief servo class to drive Lobot servos
```

Comment:
Typo, "Lobot" -> "Robot"

Reply:
no lol, Lobot is the brand of the servos:
https://www.amazon.it/Lobot-lx-16-Smart-Serial-Digital/dp/B07D4HL8P3

Reply:
Oh lol, didn't know 😂